internal/services/formatters/csharp/scs/formatter.go

```diff
@@ -24,25 +25,36 @@ import (
 	"github.com/ZupIT/horusec-devkit/pkg/enums/tools"
 	"github.com/ZupIT/horusec-devkit/pkg/utils/logger"
 
-	dockerEntities "github.com/ZupIT/horusec/internal/entities/docker"
-	errorsEnums "github.com/ZupIT/horusec/internal/enums/errors"
+	"github.com/ZupIT/horusec/internal/entities/docker"
 	"github.com/ZupIT/horusec/internal/enums/images"
 	"github.com/ZupIT/horusec/internal/helpers/messages"
 	"github.com/ZupIT/horusec/internal/services/formatters"
-	"github.com/ZupIT/horusec/internal/services/formatters/csharp/scs/entities"
-	"github.com/ZupIT/horusec/internal/services/formatters/csharp/scs/enums"
-	severitiesScs "github.com/ZupIT/horusec/internal/services/formatters/csharp/scs/severities"
-	fileUtils "github.com/ZupIT/horusec/internal/utils/file"
+	fileutils "github.com/ZupIT/horusec/internal/utils/file"
 	vulnHash "github.com/ZupIT/horusec/internal/utils/vuln_hash"
 )
 
+var (
+	// ErrSolutionNotFound occurs when CsProj not found in dotnet project
+	//
+	// nolint: lll
+	ErrSolutionNotFound = errors.New("security code scan failed to execute. The current working directory does not contain a solution file")
```

Comment:
The error refers to a .sln file and not a .csproj

Reply:
I just moved the variable and their docs and don't see this, good catch.

Reply:
Fixed